examples/custom-renderer-codelab/src/index.js

```diff
@@ -10,7 +10,7 @@ import {generator} from './generators/javascript';
 import {javascriptGenerator} from 'blockly/javascript';
 import {save, load} from './serialization';
 import {toolbox} from './toolbox';
-import './renderers/javascript';
+import './renderers/custom';
```

Comment:
Is this line supposed to be in this PR? Was the file already renamed in another PR?